content/_data/changelogs/weekly.yml

```diff
@@ -1534,6 +1534,34 @@
       pull: 3122
       message: >
         Developer: Implement default methods in <code>TaskListener</code> and <code>BuildListener</code> interfaces so they don't have to be implemented in subclasses.
+- version: "2.92"
+  date: 2017-11-26
+  changes:
+    - type: rfe # https://github.com/jenkinsci/jenkins/pull/3154#issuecomment-345983962
+      pull: 3154
+      issue: 48116
+      message: >
+        Revert internal change that broke assumption in <code>ruby-runtime</code> in 2.91, impacting plugins based on it.
+    - type: rfe
+      pull: 3143
+      message: >
+        Improve UI performance with long list of running builds by caching the estimated duration.
```

Comment:
If @Jimilian wants to get these fixes backported to 2.89.x, he will need to create JIRA tickets